docker/Dockerfile

```diff
@@ -40,8 +42,9 @@ RUN touch ${LOG_PATH}/django/api_app.log ${LOG_PATH}/django/api_app_errors.log \
     # this is cause stringstifer creates this directory during the build and cause celery to crash
     && rm -rf /root/.local
 
-# download yara rules and quark engine rules
-RUN api_app/analyzers_manager/repo_downloader.sh
+# download yara rules and quark engine rules if not in development mode
+RUN if [ $DEVELOPMENT = "true" ]; then echo "Skipping repo_downloader.sh in DEVELOPMENT mode"; \
```

Comment:
proof of this line working? anyway I would prefer to add a separated bash file with this because one-liners like this are not easy to read.

Reply:
@mlodic Should I upload a screenshot on imgur and share it here for the proof? Additionally, do you want me to move the condition to the ```repo_downloader``` script itself?

Reply:
yep to both! thanks!|

Reply:
The condition has been moved to ```repo_downloader.sh```